app-text/pdfminer/pdfminer-20191020.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7} pypy )
+
+inherit distutils-r1
+
+DESCRIPTION="Python tool for extracting information from PDF documents"
+HOMEPAGE="https://euske.github.io/pdfminer/ https://pypi.org/project/pdfminer/"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]"
```

Comment:
... but this package seems to require setuptools in RDEPEND, too.

Reply:
What makes you say that?

Reply:
https://setuptools.readthedocs.io/en/latest/setuptools.html#accessing-data-files-at-runtime thought I saw this in there, could be wrong.

Reply:
The link says (my bold):
>  It is strongly recommended that, if you are using data files, you **should** use the ResourceManager API of pkg_resources

But no output from:
```
pdfminer-20191020 $ grep -rn pkg_resources
```
so I don't think it is using that. However, it does solve a problem I had been thinking about for other projects, so that was a useful read. Thanks!